ansible/library/generate_golden_config_db.py

```diff
@@ -427,6 +457,12 @@ def generate(self):
         # update dns config
         config = self.update_dns_config(config)
 
+        # update router_config_mode
+        if self.generate_frr_config_mode_golden_config_db() is True:
+            metadata = config.get("DEVICE_METADATA",{}).get("localhost",{})
+            if metadata.get("docker_routing_config_mode") != "unified":
+                config["DEVICE_METADATA"]["localhost"]["docker_routing_config_mode"] = "unified"
```

Comment:
In the path of lien 455 where config is {}. The 464 will cause error.
Do you test on DUT for this update? Let's check if the /etc/sonic/golden_config_db.json is correctly updated and paste the result in PR description